contracts/LockedNORI.sol

```diff
@@ -622,11 +630,17 @@ contract LockedNORI is
   {
     TokenGrant storage grant = _grants[account];
     uint256 balance = this.balanceOf(account);
+    uint256 vestedBalance;
+    if (_hasVestingSchedule(account)) {
+        vestedBalance = grant.vestingSchedule.availableAmount(atTime);
+    } else {
+        vestedBalance = grant.grantAmount;
+    }
```

Comment:
_might_ be cleaner as

```suggestion
    uint256 vestedBalance = _hasVestingSchedule(account)
      ? grant.vestingSchedule.availableAmount(atTime)
      : grant.grantAmount;
```

Reply:
Good idea.  I always forget which languages have ternary operator and which don't!